### PR TITLE
OpenJDK instead of Oracle JDK

### PR DESCRIPTION
**Pull request by eivantsov**
@TylerJewell @riuvshin @skabashnyuk  I have not tested it with an actual installation, just verified that Java is installed and added to path:

```
/ # java -version
openjdk version "1.8.0_111"
OpenJDK Runtime Environment (IcedTea 3.2.0) (Alpine 8.111.14-r1)
OpenJDK 64-Bit Server VM (build 25.111-b14, mixed mode)

```

### CODE REVIEW

**Comment by eivantsov** [comment]
I will test and then merge. Please hold on with merging.

**Comment by eivantsov** [comment]
Tested. Codenvy started. No issues found.

File: dockerfiles/init/modules/codenvy/templates/codenvy.env.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@ JPDA_ADDRESS=<%= scope.lookupvar('codenvy::codenvy_debug_port') %>
 <% if scope.lookupvar('codenvy::codenvy_debug_suspend') == "true" %>JPDA_SUSPEND=y<% end -%>
 <% end %>
 
-JAVA_HOME=/opt/jdk
+JAVA_HOME=/usr/lib/jvm/default-jvm/jre
```
Comment (riuvshin) [inline dockerfiles/init/modules/codenvy/templates/codenvy.env.erb]: @eivantsov @skabashnyuk maybe we should change JAVA_HOME to JDK not JRE?
like `JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk/bin` I know that it starts with JRE but im not sure if we will have all options with JRE with debugging and some JVM OPTS that we set-up. WDYT guys?

File: dockerfiles/codenvy/Dockerfile [hunk 2]
```diff
@@ -15,84 +15,25 @@
...
 
-ENV JAVA_VERSION_MAJOR=8 \
-    JAVA_VERSION_MINOR=102 \
-    JAVA_VERSION_BUILD=14 \
-    JAVA_PACKAGE=server-jre \
-    JAVA_JCE=standard \
-    JAVA_HOME=/opt/jdk \
-    PATH=${PATH}:/opt/jdk/bin \
-    GLIBC_VERSION=2.23-r3 \
+ENV JAVA_HOME=/usr/lib/jvm/default-jvm/jre \
```
Comment (riuvshin) [inline dockerfiles/codenvy/Dockerfile]: same question as here https://github.com/codenvy/codenvy/pull/1692/files#r99466225

**Comment by riuvshin** [comment]
Btw @eivantsov we need exactly same changes for saas here https://github.com/codenvy/customer-saas/blob/master/dockerfiles/saas/Dockerfile

File: dockerfiles/init/modules/codenvy/templates/codenvy.env.erb (hunk 1 above)
Comment (mshaposhnik) [inline dockerfiles/init/modules/codenvy/templates/codenvy.env.erb]: +1 maven server and dugging may fail. Let's configure JDK for now.

**Comment by riuvshin** [comment]
we need to revert those changes, we have issues with openJDK

**Pull request merged**